vavr/generator/Generator.scala

```diff
@@ -1347,6 +1348,16 @@ def generateMainClasses(): Unit = {
           public static void println() {
               System.out.println();
           }
+
+          /**
+           * Using the {@code System.out.println()} will print all the objects passed as argument
+           * separating them by a space character (' ').
+           *
+           * @param objs The list of <code>Object</code> to be printed
+           */
+          public static void println(Object ...objs) {
+              println($ListType.of(objs).map(String::valueOf).collect(java.util.stream.Collectors.joining(" ")));
```

Comment:
The implementation is semantically correct but not optimal regarding the performance characteristics.

* List.of() creates a VAVR list in O(n)
* map() takes also O(n) because it creates a completely new list. The previous list elements are garbage collected.
* collect() creates a lazy Java Stream. That is a little bit late here. It would be better to call `mkString(" ")` instead

Please do it the 'lazy' way by using an Iterator. It is created in O(1) time and memory. VAVR's implementations creates a lightweight wrapper around the given array argument. The terminal operation iterates the original array.

```java
println($IteratorType.of(objs).map(String::valueOf).mkString(" "))
```

(Maybe you need the IteratorType variable in scope, if it does not exist already.)

Reply:
Sorry, you are right. I wasn't aware of the mkString method. I'm going to fix it right now.